miro_preprocessor/Makefile

```diff
@@ -26,13 +26,25 @@ xml_to_json_converter-publish: xml_to_json_converter-build $(ROOT)/.docker/publi
 		--project=xml_to_json_converter \
 		--infra-bucket=platform-infra
 
-miro_preprocessor-test: xml_to_json_converter-test
+xml_to_json_run_task-test: $(ROOT)/.docker/python3.6_ci
+	docker run \
+		-v $(ROOT)/miro_preprocessor/xml_to_json_run_task:/data \
+		-e OP=test -e FIND_MATCH_PATHS='/data/src' \
```

Comment:
For consistency with `xml_to_json_converter-test`, do we want these paths to be:

```
-v $(ROOT)/miro_preprocessor/xml_to_json_run_task/src:/data \
-e OP=test -e FIND_MATCH_PATHS='/data'
```

?

Reply:
Also, I wonder if it would be helpful to add a second variable to our Makefiles:

```make
ROOT = $(shell git rev-parse --show-toplevel)
PROJECT = $(ROOT)/miro_preprocessor
```

Reply:
Wouldn't this cause an issue if multiple projects used this variable?

Reply:
Mmm. I think it would still work – `include` just runs the Makefile straight through, so it would only last until the variable was next defined. But might get messy, so let’s leave it for now.